obspy/io/sac/tests/test_core.py

```diff
@@ -316,7 +316,7 @@ def test_undefinedB(self):
             self.assertEqual(tr2.stats.starttime.timestamp, 269596810.0)
```

Comment:
Hello, sorry for late reply.

I don't remember having written that `test_undefinedB`. In any case I agree with @jkmacc-LANL on that SAC reference time should not be changed, at least not without recomputing al the time-relative header fields (B, A, O, T0-T9).

Concerning "B", I would suggest to always write it to disk, i.e. if a user unsets B, this is recomputed and written to disk on writing a Trace object.